src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -471,19 +473,16 @@ internal BigInteger(int n, uint[]? rgu)
 
         /// <summary>
         /// Constructor used during bit manipulation and arithmetic.
-        /// When possible the uint[] will be packed into  _sign to conserve space.
+        /// When possible the uints-span will be packed into  _sign to conserve space.
```

Comment:
```suggestion
        /// When possible the ROSpan<uint> will be packed into  _sign to conserve space.
```

Reply:
Or possibly just name it `value` so it doesn't need to be changed in the future, etc.

Reply:
Done